src/components/site-form.tsx

```diff
@@ -241,8 +241,8 @@ export const SiteForm = ( {
 	if ( importExportEnabled ) {
 		return (
 			<form className={ className } onSubmit={ onSubmit }>
-				<div className="flex flex-col gap-6">
```

Comment:
This is sub-optimal and I don't like removing it and then adding it to children manually. However, if you don't do that the `Advanced settings` div gets extra space under when the settings are closed because of a new `div` needed for animation.

Alternatively, we can add a custom class on `Advanced settings` div and try adding some animation to `index.css`. 

This is open for discussion, Maybe @sejas has some other ideas? Thanks!

Reply:
@katinthehatsite , you can use a CSS class instead , something like this:
```diff
- { isAdvancedSettingsVisible && (
<label
+	className={ cx(
+		'flex flex-col gap-1.5 leading-4 p-2',
+		! isAdvancedSettingsVisible && 'hidden'
+	) }
>
```